.github/workflows/maven.yml

```diff
@@ -14,14 +14,20 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy: 
       matrix : 
```

Comment:
What do you think about adding `max-parallel: 1` under strategy so it will only run integration tests in sequence.

Pros: we can omit adding ledger suffix
Cons: github actions will take longer to execute

Reply:
I think we should keep the prefix thing, We can however considering capping the parallel tasks as well since it would protect the tests from running into a ledger limit exception,  specially when  more than one PR is raised almost at the same time. I would revisit this later when we start facing this problem for real.